services/core-web/src/components/mine/Permit/PermitConditions.tsx

```diff
@@ -127,6 +131,12 @@ const PermitConditions: FC<PermitConditionProps> = ({
     "condition_category_code"
   );
 
+  const PERMIT_CONDITION_STATUS = {
+    complete: {icon: faCheckCircle, color: "#45a766", text: "Complete"},
+    in_progress: {icon: faClock, color: COLOR.violet, text: "In Progress"},
+    not_started: {icon: faBan, color: "#bbb", text:"Not Started"}
```

Comment:
I'd take a look at helpers.scss for colours. The core-web version is missing `&-success` which could be added (this shade of green has a variable of $ok-green it looks like). But I think these could be class names of color-success, color-primary, and color-gov-grey.